source/D2Game/include/GAME/Clients.h

```diff
@@ -218,7 +218,7 @@ struct D2ClientStrc
 	uint64_t nPingMovingAverage;				//0x4F8
 	uint32_t dwPingsCount;						//0x500
 	int32_t nExpLoss;							//0x504
-	uint32_t dwExpLost;							//0x508
+	uint32_t dwLocale;							//0x508
```

Comment:
as a version note: 1.13c+ (and likely from 1.11+) has +0x508 storing the exp loss, +0x50C as the LocaleId.

Reply:
There is also langid below, hence I'm a bit confused.

Reply:
It means that they just shifted down 4 bytes in later versions due to a new member. Blizzard tended to intermix LangId and LocaleId (they are the same thing, almost, the client binaries refer to it via the ELANGUAGE  enum sometimes, othertimes as the LangId, however all the BNet infrastructure refers to it as the client product Locale).